net/table_net_certificate.go

```diff
@@ -159,7 +160,11 @@ func tableNetCertificateList(ctx context.Context, d *plugin.QueryData, h *plugin
 			plugin.Logger(ctx).Error("net_certificate.tableNetCertificateList", "failed to perform TLS handshake:", err)
 			return nil, nil
 		}
+		if strings.HasSuffix(err.Error(), ": no such host") {
+			return nil, nil
+		}
```

Comment:
@bdd4329 Instead of checking the error string directly, can you please check the error instance to validate whether the error is due to a timeout or an invalid host? For example:
```suggestion
        // Return nil, if the given host couldn't be found
        if opErr, ok := err.(*net.OpError); ok {
	        if dnsError, isDnsError := opErr.Err.(*net.DNSError); isDnsError {
		        if dnsError.IsNotFound {
			        plugin.Logger(ctx).Error("net_certificate.tableNetCertificateList", "failed to find the host:", err)
			        return nil, nil
		        }
	        }
        }
```

Reply:
Thank you! I've made the changes that you recommended.